content/sensu-core/1.5/reference/filters.md

```diff
@@ -406,12 +406,31 @@ Here's an example of a check definition that will be filtered if a check named `
 }
 {{< /highlight >}}
 
+Or, define a dependency by specifying the subscription and the check.
+Here's an example of a check definition that will be filtered if any check named `mysql` on any client subscribed to the `db-nodes` subscription is already alerting:
+
+{{< highlight json >}}
+{
+  "checks": {
+    "web_application_api": {
+      "command": "check-http.rb -u https://localhost:8080/api/v1/health",
+      "subscribers": ["web_application"],
+      "interval": 20,
+      "dependencies": ["subscription:db-nodes/mysql"]
+    }
+  }
+}
+{{< /highlight >}}
+
+_WARNING: Specifying a subscription/check pair in the check dependencies filter
+may impact Sensu's performance in large-scale installations._
```

Comment:
Is there any way to be more specific here? Ideally by providing rough figures or guidance on when you might want to avoid using this.